kitsune/notifications/tests/test_api.py

```diff
@@ -80,7 +81,7 @@ def test_correct_fields(self):
         eq_(serializer.data['action_object']['type'], 'question')
         eq_(serializer.data['action_object']['id'], q.id)
         eq_(serializer.data['target'], None)
-        eq_(type(serializer.data['timestamp']), datetime)
+        datetime.strptime(serializer.data['timestamp'], '%Y-%m-%dT%H:%M:%SZ')
```

Comment:
In DRF 2.x, serialized datetimes were still datetimes. In DRF 3.x, datetimes get serialized to string encoded versions. This shouldn't affect the JSON rendered output, just the intermediary step.